config/samples/aigateway.yaml

```
@@ -100,7 +100,7 @@ spec:
         spec:
           containers:
           - name: controller
-            image: kong/kubernetes-ingress-controller:3.1
+            image: kong/kubernetes-ingress-controller:3.1.2
```

```suggestion
            image: kong/kubernetes-ingress-controller:3.1.4
```
Why not?

The same for others

---

'cause I wanted to stick with the defaults, but yeah sure. I've tested with 3.1.4 and it works just fine 👍